core/templates/pages/contributor-dashboard-page/services/contribution-opportunities.service.ts

```diff
@@ -26,12 +26,12 @@ import { ExplorationOpportunitySummary } from 'domain/opportunity/exploration-op
 import { LoginRequiredModalContent } from 'pages/contributor-dashboard-page/modal-templates/login-required-modal.component';
 import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
 
-export class SkillOpportunitiesDict {
+export interface SkillOpportunitiesDict {
```

Comment:
Why this change?

Reply:
It doesn't make sense to define it as a class. There are no constructors or functions it just contains behaviour that a class implements